core/node/consensus/src/storage/mod.rs

```diff
@@ -69,3 +77,101 @@ impl PayloadQueue {
         Ok(())
     }
 }
+
+#[derive(Debug, Default)]
+#[allow(dead_code)]
+pub struct CommitteeValidator {
```

Comment:
Please move the types which represent the contract API to a separate module.

Reply:
This ended up moving everything to `consensus/storage/registry_contract`.